netbox/dcim/filtersets.py

```diff
@@ -1462,17 +1462,16 @@ class InterfaceFilterSet(
     PathEndpointFilterSet,
     CommonInterfaceFilterSet
 ):
-    # Override device and device_id filters from DeviceComponentFilterSet to match against any peer virtual chassis
-    # members
-    device = MultiValueCharFilter(
-        method='filter_device',
-        field_name='name',
-        label=_('Device'),
+    virtual_chassis_for_device = django_filters.ModelMultipleChoiceFilter(
```

Comment:
This name is a bit confusing IMO. How about something like `virtual_chassis_member`?

Reply:
That works, I can get that fixed right away.